classlib/src/main/java/org/teavm/classlib/java/lang/reflect/TField.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright 2017 Jaroslav Tulach.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.teavm.classlib.java.lang.reflect;
+
+import org.teavm.classlib.java.lang.TSecurityException;
+
+
+public class TField {
```

Comment:
That's strange. From what version did you fork? Currently, there's `TField` implementation in TeaVM.

Reply:
Probably some outdated version. I've just merged with `origin/master`, so hopefully the diff looks more sane.